Gemfile

```diff
@@ -1,7 +1,7 @@
 source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-ruby '2.7.5'
+ruby File.read('.ruby-version').strip
```

Comment:
Abstractions like this should be avoided as they lead to unintended sideeffects

Reply:
Thank you, @mmatongo. I will correct that.
I was following this guide: https://docs.render.com/ruby-version

Reply:
Odd that they would recommend that